source/Nevermore/Advanced/Queryable/QueryTranslator.cs

```diff
@@ -260,6 +260,11 @@ IWhereClause CreateMethodCallWhere(MethodCallExpression expression, bool invert
                     var (fieldReference, _) = GetFieldReferenceAndType(right);
                     return sqlBuilder.CreateWhere(fieldReference, invert ? ArraySqlOperand.NotIn : ArraySqlOperand.In, values);
                 }
+                else if (left!.Type == typeof(string) &&  right.Type == typeof(StringComparison))
+                {
+                    return CreateStringMethodWhere(expression, invert);
+                }
+
```

Comment:
I think it'd make more sense to handle this case inside the `CreateStringMethodWhere()` method. That would mean dropping the check for only one argument, which is fine.

Allowing the overload but ignoring the parameter is likely to lead to more head-scratching rather than less. Any idea how difficult it'd be to properly support it?

Reply:
> StartsWith and EndsWith already work ignoring case in Nevermore.

Does this mean you can specify a `StringComparison` argument and Nevermore will allow it but not use it? If that's the case, then that sounds like a bug. As a general rule, we want the query translator to throw up its hands if it comes across anything it doesn't know how to deal with properly.

Reply:
Investigating alternatives